db/change_cache.go

```diff
@@ -520,8 +520,8 @@ type cachePrincipal struct {
 	Sequence uint64 `json:"sequence"`
 }
 
-func (c *changeCache) Remove(collectionID uint32, docIDs []string, startTime time.Time) (count int) {
-	return c.channelCache.Remove(collectionID, docIDs, startTime)
+func (c *changeCache) Remove(ctx context.Context, collectionID uint32, docIDs []string, startTime time.Time) (count int) {
+	return c.channelCache.Remove(ctx, collectionID, docIDs, startTime)
```

Comment:
Is there a pattern/rule to follow for usage of parameter context vs. `c.logCtx`? Usage-dependent?

Having context defined only in some methods makes me wonder how we're going to maintain this with any consistency going forwards.